### ISSUE TITLE
Is there a way to do this with a package manager besides conda?

### ISSUE THREAD

**Issue by bbrewington** [gnilrets/dbt-container-skeleton#12]
I'm biased toward venv, and any time I see conda mentioned, I cringe a bit since it involves installing a ton of stuff I don't need

**Comment by gnilrets** [comment]
Yeah, nothing to prevent you from using venv if that’s what you prefer. 

But you might be thinking of the full anaconda environment , which does install a lot of extra stuff.  I recommend miniconda, which only includes a fairly minimal set of packages and in my opinion makes it a bit easier to manage multiple environments with different python versions.
